src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -295,6 +300,44 @@ double Camera::extract_near_z() const
     return near_z;
 }
 
+// Assumed shutter opening/closing curve is straight line.
+float Camera::integrate_inverse_curve(float a, float b, float x) const
+{
+    float constant;
+    int root_sign;
+    // opening
+    if(b == 0){
```

Comment:
Opening and closing braces always go on their own lines.

Reply:
Insert a space between `if` and the opening parenthesis.

You can find a detailed style guide here:
https://github.com/appleseedhq/appleseed/wiki/Coding-Philosophy-and-Guidelines#coding-style